### PR TITLE
Making install call not dir dependent

### PR DESCRIPTION
**Pull request by bannsec**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](../DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](../DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the documentation and the [radare2 book](../../radare2book) with the relevant information (if needed)

**Detailed description**
When I attempted to build sanitize version previously I got an error about not finding sys/install.sh. Looks like there's an assumption that sanitize is called from the root dir. I've removed this assumption so it can be called from anywhere.

**Test plan**
```bash
cd sys
./sanitize.sh # This should work
```

**Closing issues**

### CODE REVIEW

**Comment by radare** [comment]
all the scripts in sys/ are suposed to be executed from the root directory of r2

File: sys/sanitize.sh [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # SANITIZE="address signed-integer-overflow"  # Faster build
 SANITIZE=${SANITIZE:="address undefined signed-integer-overflow"}
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```
Comment (radare) [inline sys/sanitize.sh]: this is an ugly bashism

Comment (bannsec) [reply sys/sanitize.sh]: It's ugly, but it's the most cross platform solution in bash to reliably get the working directory of the script. I have yet to have it fail on me.

**Comment by bannsec** [comment]
Eh, prolly just best to leave it as it is *shrug*

**Comment by radare** [comment]
nope. its good to fix it, but not using bashisms :)

**Pull request closed without merging**